### PR TITLE
Chip component

### PR DESCRIPTION
**Pull request by katarzynatobis**
A brand new Chip component!

Screenshot 2023-04-11 at 16 51 04

### CODE REVIEW

File: src/components/chip/ChipGroup.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+      screen.getByRole('radiogroup', {
+        description: 'ChipGroup description',
+      })
+    ).toBeInTheDocument();
+  });
+
+  describe('single select', () => {
+    it('renders single select chip group with chips', () => {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const chipGroup = renderChipGroup({});
```
Comment (clxandstuff) [inline src/components/chip/ChipGroup.spec.tsx:77]: You don't have to assign result of render to variable

**Review by clxandstuff — CHANGES_REQUESTED** [review]

File: src/components/chip/ChipGroup.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+      expect(screen.getByRole('radiogroup')).toBeInTheDocument();
+      expect(screen.getByLabelText(chips[0])).not.toBeChecked();
+      expect(screen.getByLabelText(chips[1])).not.toBeChecked();
+    });
+
+    it('changes checked element when Chip is clicked and has only one checked Chip at a time', () => {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const chipGroup = renderChipGroup({
```
[the remark this answers is not in the document]
Comment (clxandstuff) [reply src/components/chip/ChipGroup.spec.tsx:86]: When disabled, chip should not have press effect

https://user-images.githubusercontent.com/25419830/231428799-06c9ef40-4199-4407-9c02-0016314bfde6.mov

[the remark this answers is not in the document]
Comment (clxandstuff) [reply src/components/chip/ChipGroup.spec.tsx:86]: In Firefox cross is misaligned.

Screenshot 2023-04-12 at 12 17 18

**Comment by clxandstuff** [comment]
Code looks good. The only thing left is this misaligned cross on Firefox.

**Review by clxandstuff — COMMENTED** [review]
Code looks good. The only thing left is this misaligned cross on Firefox.

**Comment by katarzynatobis** [comment]
@clxandstuff could you check now?

**Comment by clxandstuff** [comment]
@katarzynatobis now it works, good job! 😄

**Review by clxandstuff — APPROVED** [review]

**Pull request merged**